.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+blank_issues_enabled: true
+contact_links:
+    about: Ask questions and discuss with other scipy community members
```

Comment:
What is this pointing to? For scikit-learn it seems to point to GitHub Discussions, but we don't have those.

Reply:
But what is it pointing to?

Reply:
Oh right it's a leftover, good catch. Thought it was the text for the empty which is the last in this list.